app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,37 @@
+module StripeWrapper
+  class Charge
+    attr_reader :status, :response
+
+    def initialize(response, status)
+      @response = response
+      @status = status
+    end
+
+    def self.create(options={})
+      StripeWrapper.set_api_key
```

Comment:
Except of having this set here, you could create `config/initializers/stripe.rb` and have it defined there when the app is started :). You would not have to set it in tests as well.